dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/icon/IconController.java

```diff
@@ -130,15 +134,29 @@ public void getIconData(@PathVariable String key, HttpServletResponse response)
   }
 
   @GetMapping
-  public @ResponseBody List<ObjectNode> getAllIcons(IconRequestParams iconRequestParams)
+  public @ResponseBody PaginatedIconResponse getAllIcons(IconRequestParams iconRequestParams)
       throws BadRequestException {
 
     IconOperationParams iconOperationParams = iconRequestParamsMapper.map(iconRequestParams);
 
+    Pager pager = null;
+
+    if (iconRequestParams.isPaging()) {
+      long total = iconService.count(iconOperationParams);
+
```

Comment:
Newlines IMHO are for grouping things that belong together. I would use them more sparingly and remove at least this one.

Reply:
i trust on our code formatter for these things.

Reply:
The code formatter does not add such newlines and will also not remove them. So you will need to adjust vertical whitespace.